README.md

```diff
@@ -117,6 +117,17 @@ The latest theme is Sink, but it relies on installing a font first which you
 can get here: https://www.dafont.com/bebas.font
 ![Sink](https://imgur.com/mzLQshj.png)
 
+### Can I fix the indicator centrally, like classic Commodore Amiga trackers?
+Yes. If you prefer that the current position is always in the center of the screen,
+similar to classic Amiga trackers such as OctaMED, enable "Fix indicator position
+in view" in the options menu.
+
+This mode is best used with "Bigger play indicator" and "Follow song" enabled.
+
+It doesn't have to be positioned centrally either. You may position the indicator
+wherever you like by holding down CTRL + ALT + LMB in the row indicator column.
+![Fix indicator in view](https://imgur.com/Ok4HG3H.gif)

```

Comment:
I don't know whether you'd prefer the animated gif or a static screenshot, but I've done both. Just need to drop this commit if you prefer the latter.

Reply:
In fact... looking at that gif, it reveals there is something not quite right, like the fixed position in edit vs play mode are slightly different. You see the line skip on the first frame. Hadn't noticed that. Probably a trivial fix, rounding error or something. I can have a look at that this evening. Not a showstopper though, if you'd rather get this through.

Reply:
Yeah, can do it as a follow-up if you want to do a fix-up later. I think the current behavior is already going to be really useful to many who are used to the commodore workflow. I'm pretty sure I'll start using it myself :shipit: 

Thanks a lot! :)

Reply:
Ha, that's the ultimate endorsement!